applications/smart-distancing/libs/loggers/csv_processed_logger.py

```diff
@@ -53,34 +54,24 @@ def log_objects(self, objects_list, distances, file_path):
 
         """
 
-        violating_objects = self.extract_violating_objects(distances)
+        violating_objects = extract_violating_objects(distances, self.dist_threshold)
+        # Get the number of violating objects (people)
+        no_violating_objects = len(violating_objects)
+        # Get the number of detected objects (people)
+        no_detected_objects = len(objects_list)
+        # Get environment score
+        environment_score = mx_environment_scoring_consider_crowd(no_detected_objects, no_violating_objects)
```

Comment:
Since we have two methods to determine the environment score maybe it was better if you would added it as a parameter of config file and run corresponding method based on user choice.

Reply:
I keep it for future work. Because it's better to think about config files and create a separate configs for some modules